amgut/lib/locale_data/american_gut.py

```diff
@@ -330,158 +331,218 @@
+<p>
+You may be withdrawn from the study if you do not complete the consent. You may also be withdrawn from the study if you do not follow the instructions given you by the study personnel.
+</p>
 
-<p style='font-weight: bold;'>Will you be compensated for participating in this study?</p>
+<p style='font-weight: bold;'><i>Will you be compensated for participating in this study?</i></p>
 <p>You will not be financially compensated in this study.</p>
 
-<p style='font-weight: bold;'>What if you are injured as a direct result of being in this study?</p>
+<p style='font-weight: bold;'><i>What if you are injured as a direct result of being in this study?</i></p>
 <p>If you are injured or become ill as a direct result of this research study, you will be provided with medical care.</p>
 
-<p style='font-weight: bold;'>What about your confidentiality?</p>
-<p>Research records will be kept confidential to the extent allowed by law. All data about you that is entered on the survey is stored on a password-protected server located at the SDSC card access controlled facility at UCSD. The code key (that records which barcode or sample name was on your sample) is stored on a separate password-protected server that is accessible only to Professor Knightand Dr Gail Ackermann.  All analysis is done on data that has no record of who you are.  We will put the data into a place where other reseachers can access it, but there will not be a way of determining who you are from what is in there.</p>
+<p style='font-weight: bold;'><i>What about your confidentiality?</i></p>
+<p>
+Research records will be kept confidential to the extent allowed by law. All data about you that is entered on the web site is stored on a password-protected server located at the SDSC (San Diego Supercomputer Center) a biometric feature controlled-entrance facility at UCSD. The code key (that records which barcode was on your sample) is stored on a separate password-protected server that is accessible only to Professor Knight,  the sample coordinator and the database coders.  All analysis is done on data that has no record of who you are.  We will put the data into a place where other reseachers can access it, but there will not be a way of determining who you are from what is in there, e.g. we will not publish your address.  Research records may be reviewed by the UCSD Institutional Review Board.
+</p>
 
-<p>Research records may be reviewed by the UCSD Institutional Review Board.</p>
 
-<p style='font-weight: bold;'>Who can you call if you have questions?</p>
-<p>Dr Rob Knight and/or Gail Ackermann has explained this study to you and answered your questions. If you have other questions or research-related problems, you may reach Dr Knight at 858-246-1184</p>
+<p style='font-weight: bold;'><i>Who can you call if you have questions?</i></p>
+<p>
+If you have questions or research-related problems, you may reach Rob Knight at 858-246-1184 or contact the American Gut Project at 858-534-8739.
```

Comment:
Nico, would it be possible to explicitly verify these phone numbers are correct with Sarah and/or Michiko?

Reply:
No problem. I will double check with them.

Reply:
thank you!